Tiltfile

```
@@ -1,8 +1,8 @@
-# Build a custom elastic-agent image with a locally built apm-server binary injected.
 custom_build(
-  'elastic-agent',
-  'bash ./testing/docker/elastic-agent/build.sh -t $EXPECTED_REF',
-  ['go.mod', 'go.sum', 'Makefile', '*.mk', '.git', 'cmd', 'internal', 'x-pack'],
+  'apm-server',
+  'docker build -t $EXPECTED_REF -f packaging/docker/Dockerfile .',
```

```suggestion
  'DOCKER_BUILDKIT=1 docker build -t $EXPECTED_REF -f packaging/docker/Dockerfile .',
```

I needed to add this to get the apm-server image to build. I think you have a newer version of `docker` which enables this by default?

---

Oh I do, it looks like that's the default starting from docker 23: https://github.com/moby/moby/releases/tag/v23.0.0

I think the env is fine, this is what we were doing for the previous approach anyway: https://github.com/elastic/apm-server/blob/62ec35461cfb8de67ae57e8bc359c9e9fb04fbe0/testing/docker/elastic-agent/build.sh#L15